Floyd_Warshall_Algorithm/Floyd_Warshall.js

```diff
@@ -0,0 +1,87 @@
+   /*The Floyd Warshall Algorithm is for solving the All Pairs Shortest Path problem.
+   The problem is to find shortest distances between every pair of vertices in a given
+   edge weighted directed Graph.This is Floyd Warshall algorithm in java script.*/
+
+   class Graph {
```

Comment:
Please remove the extra space at the start of line. Each block inside function or class or conditional statement should be indented 4 spaces. However this 3 space indentation at start of line here is unnecessary.